realm/realm-library/src/objectServer/java/io/realm/permissions/PermissionChange.java

```diff
@@ -18,6 +18,7 @@
 import java.util.Date;
 import java.util.UUID;
 
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
```

Comment:
This may require clients to include the FB annotations as a `provided` dep in their projects, to avoid warnings.